### DISCUSSION TITLE
Changing configuration folder (.smplayer folder) location in Windows 10

### DISCUSSION THREAD

**Discussion by Anony400** [smplayer-dev/smplayer#868] [Q&A]
Is there any way of changing configuration folder (.smplayer folder) location in Windows 10?

If so, please detail how to do it (not the most technical).

**Comment by smplayer-dev** [comment]
I didn't tested on Windows, but the configuration folder can be specified with the command-line parameter `-config-path`

**Comment by Anony400** [comment]
How do I use command line for windows 10?

**Comment by smplayer-dev** [comment]
You can edit the smplayer shortcut and add the -config-path option (type after it the path of the folder).
This explains how to add a parameter to a shortcut:
https://www.digitalcitizen.life/shortcut-arguments-parameters-windows/
